.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
I added this to [configure the version of Node](https://docs.netlify.com/configure-builds/manage-dependencies/#node-js-and-javascript) used in our Netlify builds.  If we don't want this file in the repo, we can instead set a `NODE_VERSION` env var before building the site.

A potential benefit of having this file here is to set the version of Node for others that use `nvm`.